api/testdata/README.md

```diff
@@ -1,3 +1,3 @@
-Current contracts manually generated from commit `v0.14.0-beta1`
+Current contracts manually generated from commit `v0.14.0-beta2`
 of https://github.com/CosmWasm/cosmwasm. Using
-[these instructions](https://github.com/CosmWasm/cosmwasm/blob/v0.14.0-beta1/contracts/README.md#optimized-builds).
+[these instructions](https://github.com/CosmWasm/cosmwasm/blob/v0.14.0-beta2/contracts/README.md#optimized-builds).
```

Comment:
`./download_releases.sh v0.14.0-beta2` works again now

Reply:
Yup, that's how I got them.